Tests/Tests/CrashLoggingTests.swift

```diff
@@ -223,6 +225,7 @@ private class MockCrashLoggingDataProvider: CrashLoggingDataProvider {
 
 /// Use this to get a `CrashLoggingDataProvider` implementation that allows testing the default values set via protocol extenstion.
 private struct CrashLoggingDataProviderWithDefaultValuesOnly: CrashLoggingDataProvider {
+    var errorEventsSamplingRate: Double
```

Comment:
Following up on the comment above regarding testing that the default value for `errorEventsSamplingRate` is `1.0`, this configurable property is not necessary.

```suggestion
/// Use this to get a `CrashLoggingDataProvider` implementation that allows testing the default values set via protocol extenstion.
private struct CrashLoggingDataProviderWithDefaultValuesOnly: CrashLoggingDataProvider {
```

Reply:
Thank you Gio!!